README.md

```diff
@@ -11,7 +11,33 @@ If you're interested in the research network, see our [pages](https://network-di
 more information. Or get in touch with one of the network members.
 
 
-## How to Contribute to the Pages
+## How to Contribute to the Pages (Step by Step)
+
+The basis of the pages are markup-files. If you want to understand how markup files are rendered as a web page, see the technical explanation below.

```

Comment:
```suggestion
The basis of the pages are markdown files. If you want to understand how markdown files are rendered as a web page, see the technical explanation below.
```

Reply:
technically, markdown is markup, so it's not wrong... but mixing them may seem confusing, right?